tests/playbooks/deploy_vaultdr.yml

```diff
@@ -43,7 +43,7 @@
         name: cf_deploy
       vars:
         - deploy_bucket: cloud-initiatives-pipeline-bucket
-        - cf_template_url: https://raw.githubusercontent.com/cyberark/pas-on-cloud/v11.3/aws/DRVault-Single-Deployment.yaml
+        - cf_template_url: https://raw.githubusercontent.com/cyberark/pas-on-cloud/v11.3.1/aws/DRVault-Single-Deployment.yaml
```

Comment:
11.3.1? 
will be changed to 11.4 when Vault&DR will release?

Reply:
You need to release the CF for use 11.4